docs/tutorials/build-a-portfolio-website-with-react-webiny-apollo.md

```diff
@@ -0,0 +1,1048 @@
+              return (
+                <Route exact path={ '/'+ text} key={index}>
+                  <MyPages page={text} />
+                </Route>
+              )
+            })}
+          </Switch>
+        </Router>
+      </div>
+    ) 
+  }
+
+export default App;
+```
+
+The new changes are based on different routes we provide in the array in the `Switch` component. Then we call the `MyPages` component that holds all the pages we created.
+
+Let's see what happened in our Portfolio website.
+
+![Admin user](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/result-apollo-client.png)
+
+**Yay** 🎉
+
+To be able to show the blogs and projects, we need to do some small adjustments.
+
+Let's open the `index.tsx` file in the `src` folder. To make the Apollo Client work, we need to connect it to React with the `ApolloProvider`. In `index.js`, let's wrap our React app with an `ApolloProvider`. Beside the Apollo Client, in this file, we will connect our React app with the Material-UI theme that we created earlier.
+
+```tsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider } from '@material-ui/core/styles';
+import theme from './theme';
+import * as serviceWorker from './serviceWorker';
+
+// ApolloClient
+import { ApolloProvider } from "@apollo/client";
+import client from './apolloClient/index';
+
+
+ReactDOM.render(
+  (<ApolloProvider client={client}>
+    <ThemeProvider theme={theme}>
+      <React.Fragment>
+        <CssBaseline />
+        <App />
+      </React.Fragment>
+    </ThemeProvider>,
+  </ApolloProvider>),
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.unregister();
+```
+
+Let's see the changes regarding the theme we provided:
+
+![result-apollo-client](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/result-apollo-client-2.png)
+
+Now the colors have been changed 😊
+
+### 2.6 `.env` file
+
+On the root of your front-end project, create a `.env` file. This file will hold the API and the access token we will use to query our content models. Refresh the information by visiting the `Access Token` section.
+
+```
+REACT_APP_GRAPHQL_URL="https://YOUR - Content Delivery API/cms/read/production"
+REACT_APP_ACCESS_TOKEN="Your Access Token"
+```
+
+Let's test it now 🚀
+
+![result-apollo-client](/img/tutorials/build-a-portfolio-webste-with-react-webiny-apollo/result-apollo-client-3.png)
+
+Yay 🎉, you created a Portfolio Website with React, Webiny, and Apollo.
+
+
+### 2.7 Deployment
+
+One more on the completion of the Portfolio is having the project online, so you can share with potential employees, recruiters or your friends 😊
```

Comment:
We can make it a little short:

`One more on the completion of the Portfolio is having the project online to share with potential employees, recruiters, or friends. 😊`